test/shape_test.cpp

```diff
@@ -526,7 +537,7 @@ TEST_CASE(test_shape_broadcasted)
 TEST_CASE(test_shape_broadcasted2)
 {
     migraphx::shape s{migraphx::shape::float_type, {1, 2}, {0, 1}};
-    EXPECT(not s.standard());
+    EXPECT(s.standard());
```

Comment:
I think we dont allow a shape to be both standard and broadcasted as far as I've seen. Maybe problematic? @umangyadav might know better

Reply:
Agree with shiv here

Reply:
Then I could put a check for broadcasted in the same spot in code.  Is that what we want?

If a shape being both standard and broadcasted is a problem, I wonder why the change didn't cause something else to fail.

Reply:
I just found an existing test case that is both standard and broadcasted:
```
TEST_CASE(test_shape_scalar2)
{
    migraphx::shape s{migraphx::shape::float_type, {1}, {0}};
    EXPECT(s.standard());
    EXPECT(s.packed());
    EXPECT(not s.transposed());
    EXPECT(s.broadcasted());
}
```

Reply:
Hmm ok, seems like its allowed for scalars. 
Off the top of my head Im not sure where/if this is problematic. Might want to confirm with Paul when hes back